components/layout/menu.hbs

```diff
@@ -1,316 +1,293 @@
-              title="Blog"
-              role="menuitem"
-            >Blog</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/services/security"
-              title="Security"
-              role="menuitem"
-            >Security</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/communities/"
-              title="Communities"
-              role="menuitem"
-            >Communities</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/client-success/"
-              title="Client Successes"
-              role="menuitem"
-            >Client Successes</a></li>
-          <li
-            class="submenu-item is-submenu-item is-dropdown-submenu-item"
-            role="none"
-          ><a
-              href="/our-expertise/partnerships"
-              title="Partnerships"
-              role="menuitem"
-            >Partnerships</a></li>
-        </ul>
-      </li>
-      <li class="top-level-item" role="none"><a
-          title="Contact"
-          href="/contact-us/"
-          role="menuitem"
-        >Contact</a></li>
-    </ul>
-    <div id="country-selector" class="country-selector show-for-large">
-      <ul
-        class="dropdown menu simple"
-        data-dropdown-menu="guqpuq-dropdown-menu"
-        data-closing-time="500"
-        data-hover-delay="200"
-        data-alignment="right"
-        role="menubar"
-      >
-        <li class="top-level-item" role="none"><a
-            title="Login"
-            href="https://my.aquent.com/mat/myaquent?PROC=AWUIDrawMatLogin&amp;locale=en_US"
-            role="menuitem"
-          >Login</a></li>
-        <li
-          class="is-dropdown-submenu-parent opens-left"
-          role="none"
-          aria-haspopup="true"
-          aria-label="USA/Canada"
-        >
-          <a
-            class="down-arrow after"
-            title="USA/Canada"
-            tabindex="0"
-            role="menuitem"
-          >USA/Canada</a>
-          <ul
-            class="submenu menu is-dropdown-submenu first-sub vertical"
-            data-submenu=""
-            role="menubar"
-          >
-            <li role="none" class="is-submenu-item is-dropdown-submenu-item"><a
-                href="https://aquent.com.au"
-                title="Australia"
+
+  <div class="grid-container top-menu">
+    <div class="site-title">
+      <a class="logo" href="/" title="Aquent"><img
+          src="/img/Vitamin-T-Logo.svg"
+        /></a>
+    </div>
+
+    <div class="grid-container navbar" id="navbar">
+      <details>
+        <summary class="tertiary-info--large">
+          <span class="menuitem">
+            <i class="arrow--before"></i>Talents<i class="arrow--after"></i>
```

Comment:
We should swap out the `<i>` tag here for something more semantically correct. Probably just a `<span>`. And our class would be `top-menu__arrow--before`.